src/custom/hooks/useContract/index.ts

```diff
@@ -21,6 +26,16 @@ export * from './useContractMod'
 
 // Custom (non-MOD) hooks
 
+export function useEthFlowContract(): CoWSwapEthFlow | null {
```

Comment:
Since this is a new code (not override), let's move it to `cow-react/modules/swap/hooks`

Reply:
will move at the end in 1 pr :)